junction/proposals/services.py

```diff
@@ -30,3 +32,23 @@ def comment_recipients(proposal_comment):
             .all().select_related('commenter')}
     recipients.add(proposal.author)
     return recipients
+
+
+def send_mail_for_new_proposal(conference, title, author, speaker_info,
+                               url, proposal_section_id, host):
+    proposal_section = ProposalType.objects.get(pk=proposal_section_id)
+    send_to = [c.reviewer for c in
+               ConferenceProposalReviewer.objects.filter(conference=conference)]
```

Comment:
A better way to do this is `conference.reviewer_set.all()`, much shorter and fewer queries.